### ISSUE TITLE
inspect --environment should display the environment

### ISSUE THREAD

**Issue by EvanTheB** [apptainer/singularity#4026]
#2431 'fixed' inspect -e to display more of the environment. Why not display the whole environment?

Env vars from a docker build are not displayed. I don't understand the usecase where some but not all environment variables are shown.

It seems like when env is available in the image, these should print the same thing (barring the extra formatting from env):

```
singularity inspect --environment image
singularity exec -C image env
```

As an alternative, at least document what the command actually does.

**Comment by Amjadhpc** [comment]
Just to confirm , i have created a singularity image using 
Singularity version 

****_singularity version 3.4.0-1_**
The base image is taken from docker as shown here
_****Bootstrap: docker**
From: **quay.io/wtsicgp/dockstore-cgpwxs:singularity_bug_report**_****

When i run the following command on the image build using the this docker file i get this

singularity inspect --environment singularity4026.sif
==environment==
==environment:90-environment.sh==
#!/bin/sh
# Custom environment shell code should follow****


    export LANG=C.UTF-8
    export PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin_**

But the original image has the following env variable in /opt which are not passed in the inspect command 

**singularity exec dockstore-cgpwxs_singularity_bug_report.sif  env | grep /opt
PATH=/opt/wtsi-cgp/bin:/opt/wtsi-cgp/biobambam2/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
PERL5LIB=/opt/wtsi-cgp/lib/perl5
OPT=/opt/wtsi-cgp**



The same thing i have tested with a singularity image  from bootstrap local .

Ideally i would like to know which env variable i am inheriting from base/parent  image

**Comment by cclerget** [comment]
Fixed by #5130
